pilot/docker/Dockerfile.proxyv2

```diff
@@ -8,7 +8,7 @@ RUN curl -s http://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -
 RUN echo "deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-7 main" \
     >/etc/apt/sources.list.d/llvm.list
 RUN apt-get update && apt-get install --no-install-recommends -y \
-    libc++-7-dev libc++abi-7-dev
+    libc++-8-dev libc++abi-8-dev
```

Comment:
why do we need this here but not on master?

Reply:
release-1.3+ is statically linked against libc++, so we don't need libc++ installed system-wide. Older releases are using older versions of Bazel, which didn't allow to do that (at least not trivially, and we were working against the deadline for HTTP/2 CVEs), so we ended up linking it dynamically in those older releases... Temporary glitch.